doc/recipe/preprocessor.rst

```diff
@@ -1196,14 +1354,17 @@ looking at the DJF field, but your datasets starts on January 1st, the first
 DJF field will only contain data from January and February.
 
 We recommend using the extract_time to start the dataset from the following
-December and remove such biased initial datapoints.
+December and remove such biased initial data points.
 
 Parameters:
-    * operator: operation to apply. Accepted values are 'mean',
-      'median', 'std_dev', 'min', 'max', 'sum' and 'rms'. Default is 'mean'
-
-    * seasons: seasons to build statistics.
-      Default is '[DJF, MAM, JJA, SON]'
+    * `operator`: Operation to apply.
+      See :ref:`stat_preprocs` for more details on supported statistics.
+      Default is `mean`.
+    * `seasons`: Seasons to build statistics.
+      Default is ``'[DJF, MAM, JJA, SON]'``.
```

Comment:
does it accept other seasons too?

Reply:
Yes, this is described in more detail in the API doc: https://esmvaltool--2191.org.readthedocs.build/projects/ESMValCore/en/2191/api/esmvalcore.preprocessor.html#esmvalcore.preprocessor.seasonal_statistics. [I didn't change anything here apart from the formatting]

Reply:
yeah got that, was just being nit-picky :grin: